raw-data/EA/EA_IPA_correspondences.tsv

```diff
@@ -1,1487 +1,1487 @@
+iɐ	iɐ	NA
+iɑ	iɑ	guess
+iɑu	iɑu	NA
+ie	ie	NA
+iẽ	iẽ	NA
+ie̞	ie̞	NA
+ĩẽ	ĩẽ	NA
+i̯e	i̯e	NA
+i̯ẽ	i̯ẽ	NA
+iẽː	iẽː	NA
+iə	iə	NA
+iə̯	iə̯	NA
+iɛ	iɛ	NA
+iɛ̃	iɛ̃	NA
+iɛ̠	iɛ̠	NA
+i̯ɛ	i̯ɛ	NA
+iɛː	iɛː	NA
+ĩɛ̃ː	ĩɛ̃ː	NA
+iɛi̯	iɛi̯	NA
+iɜ	iɜ	NA
+ii	ii	NA
+ii̯	ii̯	NA
+iɪ̯	iɪ̯	NA
+ij	ii̯	conventionalized
+io	io	NA
+iø	iø	NA
+ĩõ	ĩõ	NA
+iou	iou	NA
+iɔ	iɔ	NA
+iu	iu	NA
+iu̯	iu̯	NA
+i̯uo	i̯uo	NA
+i̯ũo	i̯ũo	NA
+iw	iu	guess
+iˤ	iˤ	NA
+iˤː	iːˤ	NA
+iˤæ	iˤæ	NA
+ɪ	ɪ	NA
+ɪ̆	ɪ̆	NA
+ɪ̈	ɪ̈	NA
+ɪ̞̈	ɪ̞̈	NA
+ɪ̃	ɪ̃	NA
+ɪ̞	ɪ̞	NA
+ɪ̯	ɪ̯	NA
+ɪ̰	ɪ̰	NA
+ɪː	ɪː	NA
+ɪ̃ː	ɪ̃ː	NA
+ɪ̯ä	ɪ̯ä	NA
+ɪə	ɪə	NA
+ɪ̃ə̃	ɪ̃ə̃	NA
+ɪɛ	ɪɛ	NA
+ɪ̃ɛ	ɪ̃ɛ	NA
+ɪɛː	ɪɛː	NA
+ɪ̃ɛː	ɪ̃ɛː	NA
+ɪ̯ɔ	ɪ̯ɔ	NA
+ɪu	ɪu	NA
+ɪ̯u̞	ɪ̯u̞	NA
+ɨ	ɨ	NA
+ɨ̆	ɨ̆	NA
+ɨ̞	ɨ̞	NA
+ɨ̠	ɨ̠	NA
+ɨ̞̰	ɨ̰̞	NA
+ɨ̤	ɨ̤	NA
+ɨ̹	ɨ̹	NA
+ɨː	ɨː	NA
+ɨ̞̃ː	ɨ̞̃ː	NA
+ɨ̟ː	ɨː	conventionalized; TODO: investigate
+ɨ̞ː	ɨ̞ː	NA
+ɨ̹ː	ɨ̹ː	NA
+ɨa	ɨa	NA
+ɨä̠	ɨä̠	NA
+ɨə	ɨə	NA
+ɨɛ	ɨɛ	NA
+ɨi	ɨi	NA
+ɨi̯	ɨi̯	NA
+ɨj	ɨi̯	conventionalized
+j	j	NA
+j̊	j̊	NA
+j̃	j̃	NA
+jː	jː	NA
+ja	i̯a	conventionalized
+jaː	i̯aː	conventionalized
+jæ	i̯æ	conventionalized
+jæi	i̯æi	conventionalized
+jai	i̯ai	conventionalized
+jau	i̯au	conventionalized
+jau̯	i̯au̯	conventionalized
+je	i̯e	conventionalized
+jẽ	i̯ẽ	conventionalized
+jei̯	i̯ei̯	conventionalized
+jɛ	i̯ɛ	conventionalized
+jɛ̃	i̯ɛ̃	conventionalized
+jɛi	i̯ɛi	conventionalized
+jɛi̯	i̯ɛi̯	conventionalized
+jo	i̯o	conventionalized
+joː	i̯oː	conventionalized
+joi	i̯oi	conventionalized
+jɔ	i̯ɔ	conventionalized
+ju	i̯u	conventionalized
+juː	i̯uː	conventionalized
+juaː	i̯uaː	conventionalized
+jui	i̯ui	conventionalized
+ʝ	ʝ	NA
+ʝː	ʝː	NA
+ɟ	ɟ	NA
+ɟ̟	ɟ̟	NA
+ɟʰ	ɟʰ	NA
+ɟʱ	ɟʱ	NA
+ɟʝ	ɟʝ	NA
+ɟʝ̞	ɟʝ̞	NA
+ɟ̥ʝ̥	ɟ̥ʝ̥	NA
+ʄ	ʄ	NA
+k	k	NA
+k˞	k˞	NA
+kː	kː	NA
+kːʰ	kːʰ	NA
+kːʷ	kːʷ	NA
+kːʷʰ	kːʰʷ	NA
+kʰ	kʰ	NA
+kʰ˞	k˞ʰ	NA
+kʰʲ	kʰʲ	NA
+kʰʷ	kʰʷ	NA
+kʰʷʲ	kʰʲʷ	new segment type added
+kʲ	kʲ	NA
+kʲː	kːʲ	NA
+kʲʰ	kʰʲ	NA
+kʲʼ	kʲʼ	NA
+kʷ	kʷ	NA
+kʷʰ	kʰʷ	NA
+kʷʲ	kʲʷ	new segment type added
+kʷʼ	kʷʼ	NA
+kx	kx	NA
+kxːʰ	kxːʰ	NA
+kxːʼ	kxːʼ	NA
+kʼ	kʼ	NA
+kʼː	kːʼ	NA
+kʼːʷ	kːʷʼ	NA
+kʼʲ	kʲʼ	NA
+kʼʷ	kʷʼ	NA
+kʼʷʲ	kʲʷʼ	new segment type added
+kˤ	kˤ	NA
+l	l	NA
+l̩	l̩	NA
+l̺	l̺	NA
+l̻	l̻	NA
+l̪	l̪	NA
+l̪̊	l̪̊	NA
+l̥	l̥	NA
+lː	lː	NA
+l̥ː	l̥ː	NA
+lˠ	lˠ	NA
+l̪ˠ	l̪ˠ	NA
+lʰ	lʰ	NA
+lʱ	lʱ	NA
+l̪ʱ	l̪ʱ	NA
+lʲ	lʲ	NA
+l̪ʲ	l̪ʲ	NA
+l̠ʲ	l̠ʲ	NA
+l̥ʲ	l̥ʲ	NA
+lʲː	lːʲ	NA
+l̥ʲː	l̥ːʲ	NA
+lˤ	lˤ	NA
+ɫ	ɫ	NA
+ɫ̪	ɫ̪	NA
+ɫ̻	ɫ̻	NA
+ɫ̺	ɫ̺	NA
+ɫ̥	ɫ̥	NA
+ɫː	ɫː	NA
+ɬ	ɬ	NA
+ɬ̪	ɬ̪	NA
+ɬ̺̪	ɬ̪̺	NA
+ɬː	ɬː	NA
+ɬˠ	ɬˠ	NA
+ɬʲ	ɬʲ	NA
+ɬʷ	ɬʷ	NA
+ɬʼ	ɬʼ	NA
+ɭ	ɭ	NA
+ɭ̥	ɭ̥	NA
+ɮ	ɮ	NA
+ɮ̺	ɮ̺	NA
+ɮʲ	ɮʲ	NA
+ɮʼ	ɮʼ	NA
+ʎ	ʎ	NA
+ʎ̟	ʎ̟	NA
+m	m	NA
+m̩	m̩	NA
+m̥	m̥	NA
+m 	m	conventionalized
+mː	mː	NA
+m̥ː	m̥ː	NA
+mˠ	mˠ	NA
+mʰ	mʰ	NA
+mʱ	mʱ	NA
+mʲ	mʲ	NA
+m̥ʲ	m̥ʲ	NA
+mʲː	mːʲ	NA
+m̥ʲː	m̥ːʲ	NA
+mʷ	mʷ	NA
+mˤ	mˤ	NA
+n	n	NA
+n̩	n̩	NA
+n̪	n̪	NA
+n̻	n̻	NA
+n̺	n̺	NA
+n̪̊	n̪̊	NA
+n̺̪	n̪̺	NA
+n̪̩	n̪̩	NA
+n̪̥	n̪̥	NA
+n̥	n̥	NA
+n̥̪	n̪̥	NA
+nː	nː	NA
+n̥ː	n̥ː	NA
+ⁿb	mb	conventionalized
+ⁿc	ɲc	conventionalized
+ⁿcʰ	ɲcʰ	conventionalized
+ⁿd	nd	conventionalized
+ⁿd̪	n̪d̪	conventionalized
+ⁿdz	ndz	conventionalized
+ⁿd͇z͇	n͇d͇z͇	conventionalized
```

Comment:
This is a case where the diacritic should not spread to the nasal.  It's "non-sibilant" (hence a *manner* diacritic) and I think we generally only want to match *place* diacritics.